tools/netlist/include/NetlistVisitor.h

```diff
@@ -78,9 +79,11 @@ static void connectVarToDecl(Netlist& netlist, NetlistNode& varNode,
 }
 
 static void connectVarToVar(Netlist& netlist, NetlistNode& sourceVarNode,
-                            NetlistNode& targetVarNode) {
-    netlist.addEdge(sourceVarNode, targetVarNode);
-    DEBUG_PRINT("Edge ref {} to ref {}\n", sourceVarNode.getName(), targetVarNode.getName());
+                            NetlistNode& targetVarNode, ast::EdgeKind edgeKind) {
+    auto& edge = netlist.addEdge(sourceVarNode, targetVarNode);
+    targetVarNode.edgeKind = edgeKind;
```

Comment:
It would be neater to extend the method to include edgeKind as an argument.

Reply:
`addEdge` is currently a `DirectedGraph` method, that has nothing to do with EdgeKind.
Do you want me to override this with a new `Netlist::addEdge(NetlistNode& sourceNode, NetlistNode& targetNode, ast::EdgeKind edgeKind)` ?

Reply:
It was a minor point so it's up to you. But that sounds reasonable.

Reply:
I added it anyway, learning on the way the importance of `using` when redefining functions on derived classes, and still wanting to call the base class version on the default function signature.